erdpy/cli.py

```diff
@@ -79,6 +78,41 @@ def setup_parser():
     query_parser.add_argument("--arguments", nargs='+')
     query_parser.set_defaults(func=query)
 
+    get_account_parser = subparsers.add_parser("get_account")
```

Comment:
Use dash format perhaps, instead of underline: `get-account` instead of `get_acount` and so on.

Reply:
done.